libuuid/src/gen_uuid.c

```diff
@@ -529,11 +531,10 @@ int __uuid_generate_time_cont(uuid_t out, int *num, uint32_t cont_offset)
 static int uuid_generate_time_generic(uuid_t out) {
 #ifdef HAVE_TLS
 	/* thread local cache for uuidd based requests */
-	const int			cs_min = (1<<6);
 	const int			cs_max = (1<<18);
 	const int			cs_factor = 2;
```

Comment:
Would be better to use a macro for all the constants?

Reply:
I don't understand your proposal.
The build failure with `-O0` is raised because a `static int` can't be initialized with a "variable" such as `const int`.
So I'm failing to understand how a macro could help.
Can you give an example?

Reply:
It's nothing against your patch. The patch is fine.

I'm commenting on the current code; maybe we can use macros for all the `cs_*` constant variables to make the code more consistent. I do not see why the code uses `const int cs_*`. I guess we can #define CS_* macros for all the variables, not only for cs_min.

Reply:
OK, this is more clear, I updated the PR to use `#define` for the other variables.